lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-network_manager-network_router.rb

```diff
@@ -4,5 +4,6 @@ class MiqAeServiceManageIQ_Providers_Openstack_NetworkManager_NetworkRouter < Mi
     expose :delete_network_router
     expose :add_interface
     expose :remove_interface
+    expose :delete_network_router_queue
```

Comment:
@andyvesel Same comment here as on https://github.com/ManageIQ/manageiq/pull/14516#discussion_r108824338.

What is the expected behavior with respect to the required `userid` parameter required for this method?

cc @mkanoor